originals/t.yaml

```diff
@@ -19,6 +19,20 @@
     genres:
     - Rhythm
 
+- name: Tails Adventure
+  names:
+  - テイルスアドベンチャー
```

Comment:
```suggestion
```
I think in this case it's ok to omit this name because even in its home market it was also known as tails adventure (https://ja.wikipedia.org/wiki/%E3%83%86%E3%82%A4%E3%83%AB%E3%82%B9%E3%82%A2%E3%83%89%E3%83%99%E3%83%B3%E3%83%81%E3%83%A3%E3%83%BC)